src/videojs.ads.js

```diff
@@ -113,12 +114,14 @@ var
           updateEnded = function() {
             ended = true;
           };
-        player.currentTime(snapshot.currentTime);
```

Comment:
This seems really strange to me. It may address the issue, but it doesn't feel like the right path. It seems counter-intuitive that `currentTime` would be different after an ad. Could this be exposing an issue in HLS/MSE? Has this been tried with a plain old MP4?